extension/src/experiments/webview/messages.ts

```diff
@@ -68,7 +73,7 @@ export class WebviewMessages {
     hasStages: () => Promise<string>,
     addStage: () => Promise<boolean>,
     getNumCommits: () => Promise<number>,
-    changeNbOfCommits: () => Promise<void>
+    changeNbOfCommits: (...args: (ExperimentFlag | string)[]) => Promise<void>
```

Comment:
[I] I think this function name should be updated now as it is not only changing the number of commits.

If `ExperimentsData` knows how to call the CLI (either branches or number of commits) then this can simply be the update function?

Reply:
I had a flash about this last night. Last week I had left this out for later and obviously I forgot about it (definitely not the best way to work).